qualang_tools/wirer/connectivity/connectivity_base.py

```diff
@@ -71,6 +71,19 @@ def _make_qubit_pair_elements(self, qubit_pairs: QubitPairsType):
 
         return elements
 
+    def _make_qubit_pair_element_pair(self, qubit_pairs: QubitPairsType):
+        if not isinstance(qubit_pairs, list):
+            qubit_pairs = [qubit_pairs]
+
+        elements = []
+        for qubit_pair in qubit_pairs:
+            id = QubitPairReference(*qubit_pair)
+            if id not in self.elements:
+                self.elements[id] = Element(id)
+            elements.append(self.elements[id])
+
+        return elements
+
```

Comment:
Sorry, please remove this. (this comes from before the discussion where I thought we need to add two elements under qubit_pair to drive control and target. we will drive only control based on the discussion).

Reply:
No problem. I removed that method that you added